compiler_opt/rl/saved_model_tools.py

```diff
@@ -0,0 +1,62 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tools for manipulating saved models."""
+
+import tensorflow as tf
+import os
+
+# greedy_policy is needed during conversion
```

Comment:
why this is needed during conversion? 

and we need to convert the collect_policy (so non greedy) during training

Reply:
Huh. Maybe it's an artifact of when I was incorrectly specifying how to carry out the conversion, when I was allowing undefined (TF-full defined) opcodes, and this was then required. Seems to work w/o. Removed.